src/components/Calculator.js

```diff
@@ -1,63 +1,87 @@
 import React from 'react';
+import calculate from '../logic/calculate';
+import operate from '../logic/operate';
```

Comment:
- [ ] [OPTIONAL] operate.js is already being used by calculate.js, so there's no need to import it